src/App.js

```diff
@@ -165,6 +169,9 @@ class App extends Component {
               
               {/*Tool-tip*/}
               {this.renderTooltip()} 
+              <div style={{position: 'absolute', left: 20, top: 20}}>
+                <NavigationControl showCompass={false} />
+              </div>
```

Comment:
Could these inline styles live in App.css instead?

Reply:
Removed inline styling. However, couldn't put it in App.css. 
Docs for NavigationControl suggests using  mapbox-gl stylesheet to work properly. So, followed how other react-map-gl components were styled.

Reply:
Checking this out too. I think this styling works well! Great job implementing this, Ritesh! I was interacting with it, and I feel like each time you hit one of the buttons, it zooms really far in or out. Is there a prop you can add from react-map-gl that allows you to set the distance per click to smaller than 1 zoom level (I am assuming that's what it's doing now). Let me know if you agree or not that a more subtle zoom would be better UX.

Reply:
I couldn't find specific prop to control zoom amount in react-map-gl or mapbox, each click on +/- button increase/decrease zoom level by 1 (hard coded in react-map-gl function codes).
So, instead I added a function to change the zoom level provided by react-map-gl and edit it based on our need on updateViewport.